deckhouse-controller/cmd/deckhouse-controller/start.go

```diff
@@ -177,16 +177,16 @@ func run(ctx context.Context, operator *addon_operator.AddonOperator) error {
 
 	operator.ModuleManager.SetModuleLoader(dController)
 
+	// Init deckhouse-config service with ModuleManager instance.
+	d8config.InitService(operator.ModuleManager)
+
 	err = operator.Start()
 	if err != nil {
 		return err
 	}
 
 	dController.RunControllers()
 
-	// Init deckhouse-config service with ModuleManager instance.
-	d8config.InitService(operator.ModuleManager)
-
```

Comment:
I had to move d8config.InitService() a little upper in the start sequence in order that it would be possible to update its possibleNames field as modules are getting registered at startup.